config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/beans/SsgActiveConnector.java

```diff
@@ -0,0 +1,94 @@
+package com.ca.apim.gateway.cagatewayconfig.beans;
+
+import com.ca.apim.gateway.cagatewayconfig.bundle.builder.AnnotableEntity;
+import com.ca.apim.gateway.cagatewayconfig.bundle.builder.AnnotatedEntity;
+import com.ca.apim.gateway.cagatewayconfig.bundle.builder.AnnotationDeserializer;
+import com.ca.apim.gateway.cagatewayconfig.config.spec.ConfigurationFile;
+import com.ca.apim.gateway.cagatewayconfig.config.spec.EnvironmentType;
+import com.ca.apim.gateway.cagatewayconfig.util.entity.EntityTypes;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.google.common.annotations.VisibleForTesting;
+
+import javax.inject.Named;
+
+import java.util.Map;
+import java.util.Set;
+
+import static com.ca.apim.gateway.cagatewayconfig.config.spec.ConfigurationFile.FileType.JSON_YAML;
+import static com.fasterxml.jackson.annotation.JsonInclude.Include.NON_NULL;
+
+@JsonInclude(NON_NULL)
+@Named("SSG_ACTIVE_CONNECTOR")
+@ConfigurationFile(name = "active-connectors", type = JSON_YAML)
+@EnvironmentType("SSG_ACTIVE_CONNECTOR")
+public class SsgActiveConnector extends GatewayEntity implements AnnotableEntity {
+    private String enabled;
```

Comment:
Are we storing this value? I have not seen this in other entities. And we are storing it then shouldn't this be `boolean`?

Reply:
For other entities its hardcoded to true, not sure why

Reply:
Done